src/main/java/ml/shifu/shifu/container/meta/MetaFactory.java

```diff
@@ -341,11 +342,11 @@ private static boolean isJsonIngoreField(Field field) {
         return false;
     }
 
-    static Set<String> filterSet = Sets.newHashSet(new String[] { "NumHiddenLayers", "ActivationFunc",
-            "NumHiddenNodes", "LearningRate", "DropoutRate", "RegularizedConstant", "L1orL2", "MaxDepth",
-            "MinInstancesPerNode", "MinInfoGain", "MaxStatsMemoryMB", "TreeNum", "Impurity", "FeatureSubsetStrategy",
-            "Loss", "LearningDecay", "Propagation", "GBTSampleWithReplacement", "Kernel", "Const", "Gamma",
-            "EnableEarlyStop", "ValidationTolerance", "MaxLeaves", "MaxBatchSplitSize" });
+    static Set<String> filterSet = Sets.newHashSet(new String[] { "NumHiddenLayers", "ActivationFunc", "NumHiddenNodes",
+            "LearningRate", "DropoutRate", "RegularizedConstant", "L1orL2", "MaxDepth", "MinInstancesPerNode",
+            "MinInfoGain", "MaxStatsMemoryMB", "TreeNum", "Impurity", "FeatureSubsetStrategy", "Loss", "LearningDecay",
+            "Propagation", "GBTSampleWithReplacement", "Kernel", "Const", "Gamma", "EnableEarlyStop",
+            "ValidationTolerance", "MaxLeaves", "MaxBatchSplitSize" });
 
```

Comment:
One issue I left here, can we make such parameters not configured here but read from ModelConfigMeta.json?

Reply:
@zhangpengshan I think using key string starting with train#params# should cover all train params?

Reply:
that's very good idea, thanks:)